src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1823,15 +1820,13 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
 
                     List<FeedbackResponseAttributes> responsesForThisQn;
 
-                    boolean isPrivateSessionCreatedByThisUser = session
-                            .isCreator(userEmail) && session.isPrivateSession();
+                    boolean isPrivateSessionCreatedByThisUser = session.isCreator(userEmail)
+                                                                && session.isPrivateSession();
```

Comment:
Recently we decided not to use custom indentation. Use 8 spaces but insert the line break at as high level as possible. in this case, it can be:
```java
boolean isPrivateSessionCreatedByThisUser = 
        session.isCreator(userEmail) && session.isPrivateSession();
```